src/vivarium/framework/resource.py

```diff
@@ -250,7 +255,7 @@ def _to_graph(self) -> nx.DiGraph:
                 if dependency not in self._resource_group_map:
                     # Warn here because this sometimes happens naturally
                     # if observer components are missing from a simulation.
-                    self.logger.warning(
+                    self.logger.warning(  # type: ignore[no-untyped-call]
```

Comment:
untyped from loguru. I tried using the `loguru-mypy` package but it didn't get rid of this particular error wo I just ignore it